### PR TITLE
kubevirt steps added to the documentation

### PR DESCRIPTION
**Pull request by mkarg75**

### CODE REVIEW

**Comment by mkarg75** [comment]
downloading the virtctl binary in the 03 script to /usr/loca/bin

File: chimera-docs/06_deploy_kubevirt.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+## Intro
+
+
+## Lab
+
+First we create the security context constraint and then deploy the kubevirt operator.
+```
+oc adm policy add-scc-to-user privileged -n kubevirt -z kubevirt-operator
+oc create -f chimera-kubevirt/kubevirt-operator.yaml
```
Comment (dustinblack) [inline chimera-docs/06_deploy_kubevirt.md:11]: I'm getting what looks like a syntax error from the yaml when applying this. I haven't figured out what the source of the problem is yet.

File: chimera-kubevirt/kubevirt-operator.yaml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+              port: "metrics"
+              path: "/metrics"
+            initialDelaySeconds: 5
+            periodSeconds: 10
+          env:
+            - name: OPERATOR_IMAGE
+              value: *image
+      securityContext:
+        runAsNonRoot: true
+~
```
Comment (dustinblack) [inline chimera-kubevirt/kubevirt-operator.yaml]: Ah ha. You picked up a tilde somehow that is causing the YAML syntax error.

File: chimera-docs/06_deploy_kubevirt.md (hunk 1 above)
Comment (dustinblack) [reply chimera-docs/06_deploy_kubevirt.md:11]: Found it. See chimera-kubevirt/kubevirt-operator.yaml

File: chimera-docs/06_deploy_kubevirt.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+oc apply -f role.yaml
+oc apply -f role_binding.yaml
+oc apply -f crds/kubevirt_v1alpha1_kwebui_crd.yaml
+oc apply -f operator.yaml
+oc apply -f crds/kubevirt_v1alpha1_kwebui_cr.yaml
+```
+
+Wait for the pod creation to finish, watch it using
+```
+watch oc get pods -n kubevirt
```
Comment (dustinblack) [inline chimera-docs/06_deploy_kubevirt.md]: Should this be -n kubevirt-web-ui instead?

**Pull request merged**